docs/it/metadata_oidc_rp.rst

```diff
@@ -16,46 +16,86 @@ OpenID Connect Relying Party Metadata (RP)
  }
 
 
-Dove un RP non disponesse all’interno dei propri metadati dei claim **client_registration_types** i valori da intendersi come impliciti sono i seguenti.
+Dove un RP non disponesse all'interno dei propri Metadata dei claim **client_registration_types** i valori da intendersi come impliciti sono i seguenti.
 
 .. list-table:: 
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio** 
    * - **client_registration_types**
-     - String
-     - Array che specifica i tipi supportati dalla federazione (solo *automatic*)
+     - Array di stringhe che specifica i tipi supportati dalla federazione (solo *automatic*)
      - |check-icon| 
+   * - **organization_name**
+     - Un nome leggibile che rappresenta l'organizzazione proprietaria dell'RP. 
+     - |check-icon| 
+   * - **jwks**
+     - JSON Web Key Set |br|
+       :rfc:`7517#appendix-A.1` |br|
+       È composto dai seguenti claim:
```

Comment:
rimuovere "È composto dai seguenti claim: ..."

perché sta in RFC7517 e non lo ridefiniamo qui